app/views/shell/_top.html.erb

```diff
@@ -18,13 +18,6 @@
       <% end %>
       <meta name="environment" content="<%= Rails.env %>">
       <%= render "layouts/styles" %>
-      <style>
```

Comment:
This is an old hack we can remove. ✂️

(If I'm adding a bit of a hack to fix this, best to balance the universe and remove one too)

Reply:
Can't wait do drop that `<style>` from there - its been driving me crazy ever since